drunc_ui/settings/settings.py

```diff
@@ -154,6 +154,11 @@
 
 KAFKA_ADDRESS = os.getenv("KAFKA_ADDRESS", "kafka:9092")
 
+KAFKA_TOPIC_REGEX = {
+    "PROCMAN": "^control\..+\.process_manager$",
```

Comment:
What do people think? This newer less readable style, or the risk of false matches? I can add some comments here, but the gist is that `"control.*.process_manager"` actually matches some unexpected things, such as

* `controlAprocessmanager`
* `control..process_manager`,
* `TYPOcontrol.root.process_managerTYPO`,
* ...

Reply:
Oh, no, if this regex string is the correct one, so be it. But it will be helpful, I think, to include some explanation of what it matches - or it is meant to match.

Reply:
Done.